### PR TITLE
Feature/technical expand firestore adapters

### PR DESCRIPTION
**Pull request by santidrj**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPathTest.java  1
- src/main/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPath.java  20
         

Clones removed
==============
+ src/main/java/org/pesmypetcare/webservice/dao/FreqWashDao.java  -1
+ src/main/java/org/pesmypetcare/webservice/service/FreqWashService.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5457278&bid=17888337)

File: src/main/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPath.java [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+            case freqTrainings:
+            case kcalsAverages:
+            case weekTrainings:
+                return buildThreeLevelPath(collection, ids).toString();
+            default:
+                throw new EnumConstantNotPresentException(Collections.class, collection.name());
+        }
+    }
+
+    private static void checkProvidedIds(@NonNull String[] ids) {
```
Comment (xavier-campos-diaz) [inline src/main/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPath.java:67]: You should comment this method

**Review by AlbertPG — APPROVED** [review]
Okey I approve it

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Now it is better

File: src/main/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPath.java (hunk 1 above)
Comment (santidrj) [reply src/main/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPath.java:67]: Done

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPathTest.java  1
- src/main/java/org/pesmypetcare/webservice/firebaseservice/firestore/DocumentPath.java  20
         

Clones removed
==============
+ src/main/java/org/pesmypetcare/webservice/dao/FreqWashDao.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5457278&bid=17888337)

**Pull request merged**